app/interactors/update_project/prepare_params.rb

```diff
@@ -0,0 +1,26 @@
+class UpdateProject
+  class PrepareParams
+    include Interactor
+
+    delegate :current_user, :project_params, :project_obj, to: :context
+
+    def call
+      context.prepared_project_params = prepared_project_params
+      context.project_obj = project_obj
```

Comment:
why do you add project_obj to context? It is alraedy in the context

Reply:
Fixed